src/main/java/com/chessmates/repository/MockGameRepository.groovy

```diff
@@ -14,7 +14,7 @@ class MockGameRepository implements GameRepository {
     Map store = new HashMap()
 
     @Override
-    void save(game) {
```

Comment:
When I looked through code, it seemed like we are storing whatever we are receiving from lichess API - anything after the last saved. This use case can be served by saveAll() alone. That is what I thought.